package.xml

```diff
@@ -2,67 +2,22 @@
 <package format="2">
   <name>twist</name>
   <version>0.0.0</version>
-  <description>The twist package</description>
+  <description>The TWIST package</description>
 
-  <!-- One maintainer tag required, multiple allowed, one person per tag -->
-  <!-- Example:  -->
-  <!-- <maintainer email="jane.doe@example.com">Jane Doe</maintainer> -->
+  <maintainer email="david@todo.todo">david</maintainer>
+  <maintainer email="emanuel@todo.todo">emanuel</maintainer>
+  <maintainer email="luisfiliperdias@gmail.com">luis</maintainer>
```

Comment:
Please start the names with capital letter :smile:

Reply:
done